.storybook/main.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path')
+
+module.exports = {
+  stories: [
+    './stories/**/*.stories.mdx',
+    './stories/**/*.stories.@(js|jsx|ts|tsx)'
+  ],
+  addons: [
+    '@storybook/addon-a11y',
+    '@storybook/addon-links',
+    '@storybook/addon-essentials',
+    '@storybook/addon-interactions'
+  ],
+  staticDirs: ['../public'],
+  framework: '@storybook/react',
+  core: {
+    builder: '@storybook/builder-webpack5'
+  },
+  webpackFinal: async (config, { configType }) => {
```

Comment:
I don't see the destructured `configType` being used here. Could you remove it or add a comment with what you anticipate using it for?

Reply:
Please see the comments at the line 20 - 23(right after the line 19 😃) in this file. In case, I included the `configType`  for future use. 
``` 
 // (resource) https://storybook.js.org/docs/react/builders/webpack
 //  Change the configuration based on `configType` which has a value of 'DEVELOPMENT' or 'PRODUCTION'
 // 'PRODUCTION' is used when building the static version of storybook.
```